### ISSUE TITLE
file manager form fields missing labels

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6813]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the text boxes on the `File manager` page do not have a label and do not meet WCAG AA requirements

Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin, go to https://dev.nurax.samvera.org/concern/images/ww72bb76t/file_manager?locale=en and run the Site Improve browser extension
![file_maanger_form_field](https://github.com/samvera/hyrax/assets/32644254/86a43e1d-272e-4c62-8dca-3a3e9b379a9e)

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue.
